Logger/Utils.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#ifdef LOGGER_LIB
+#ifdef _LIB
```

Comment:
1) _LIB macros is too generic. If someone include your logger to some project which has _LIB macros dllexport will be used
2) Filename: UTils.h is something that provide generic functionality for all project (e.g StringUtils, some math etc). Rename the file to LoggerExport.h or LoginGlobal.h.